lib/instant.ts

```diff
@@ -39,17 +39,17 @@ export class Instant implements Temporal.Instant {
   get epochSeconds(): Return['epochSeconds'] {
     if (!ES.IsTemporalInstant(this)) throw new TypeError('invalid receiver');
     const value = GetSlot(this, EPOCHNANOSECONDS);
-    return JSBI.toNumber(ES.BigIntFloorDiv(value, BILLION));
+    return JSBI.toNumber(BigIntFloorDiv(value, BILLION));
   }
   get epochMilliseconds(): Return['epochMilliseconds'] {
     if (!ES.IsTemporalInstant(this)) throw new TypeError('invalid receiver');
     const value = JSBI.BigInt(GetSlot(this, EPOCHNANOSECONDS));
```

Comment:
Why is value being wrapped by `JSBI.BigInt` here, but it's not in `epochSeconds` above?

Reply:
Probably a mistake. Since this line is already a diff due to JSBI usage, I'll add another commit to remove it.